### DISCUSSION TITLE
Got my first build delivered.

### DISCUSSION THREAD

**Discussion by baerrs** [OMOTE-Community/OMOTE-Firmware#66] [General]
I got five fully populated boards (minus the antenna, and it still connects via wifi)—only two with the LCD and battery.  I still need to install the battery, but the LCD is working.  I was able to build and flash the software.  I confirmed the MQTT data had been received.  

For those of you who have boards...  where should I start to get some functionality from the remote?

**Comment by KlausMu** [comment]
https://github.com/CoretechR/OMOTE/wiki/How-to-understand-and-modify-the-software
